Composer/packages/client/src/components/CreationFlow/v2/CreationFlow.tsx

```diff
@@ -146,9 +148,9 @@ const CreationFlowV2: React.FC<CreationFlowProps> = () => {
     }
   };
 
-  const handleCreateNext = async (data: string) => {
+  const handleCreateNext = async () => {
     setCreationFlowStatus(CreationFlowStatus.NEW_FROM_TEMPLATE);
-    navigate(`./create/${data}`);
+    navigate(`./create/template`);
```

Comment:
I wonder if some of this could have been avoided by doing an encodeURIcomponent on the name value...

Reply:
I confirmed this was the case.